### PR TITLE
update composer.lock file

### PR DESCRIPTION
**Pull request by xabbuh**
Someone noticed on IRC that the current lock file for 3.1.x-dev was not really usable.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -49,6 +49,8 @@
             "php": "5.5.9"
         }
     },
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```
Comment (xabbuh) [inline composer.json]: This is needed for the APCu polyfill.

**Comment by fabpot** [comment]
Thank you @xabbuh.

File: composer.json (hunk 1 above)
Comment (Pierstoval) [inline composer.json]: Isn't this really dangerous? I mean, isn't there a way to release a proper release for the APCu polyfill, even a beta/rc for the minimum stability to rise up and not be `dev`?

Comment (xabbuh) [inline composer.json]: Imo that's okay for the moment as Symfony 3.1 itself isn't stable yet. But yeah having a release for the APCu polyfill will be better.

Comment (Pierstoval) [inline composer.json]: This must not be forgotten when `3.1` will reach stabilization/release phase :confused:

Comment (fabpot) [inline composer.json]: That's how we did that in the past for previous versions.

Comment (Pierstoval) [inline composer.json]: Ok no problem for that then :)

**Pull request merged**